csrc/codegen.cpp

```diff
@@ -3560,6 +3560,11 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
     const auto buffer_dtype = alloc->buffer()->dtype();
 
     NVF_ERROR(alloc->buffer() != nullptr);
+
+    if (alloc->buffer()->isFusionOutput()) {
```

Comment:
Because `Allocate` is now created for fusion outputs

Reply:
Are we moving towards having native alias support for things like inplace-update?!
:heavy_heart_exclamation:

Reply:
`kir::Allocate` is use to represent memory-sharing (i.e., aliasing) relationships. An `Allocate` node for the output is used to share a buffer with an output tensor.

Since it is allocated outside of a CUDA kernel, there's nothing to do here. We could generate some comment line like we do for intermediate gmem tensors, but I'm not sure if that's helpful.